README.md

```diff
@@ -46,6 +48,29 @@ module.exports = {
 }
 ```
 
+If you need to use options with `css-loader`.
+
+```js
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
+
+module.exports = {
+  module: {
+    rules: [
+      {
+        test: /\.css$/,
+        use: ExtractTextPlugin.extract({
+          fallback: "style-loader",
+          use: 'css-loader?modules,localIdentName="[name]-[local]-[hash:base64:6]"'
```

Comment:
It would be better to write this as

```
use: {
  loader: 'css-loader',
  options: {
    modules: true,
    localIdentName: '[name]-[local]-[hash:base64:6]'
  }
}
```

since that reads better.